homeassistant/components/webhook/__init__.py

```diff
@@ -38,14 +40,21 @@ def async_register(
     name: str,
     webhook_id: str,
     handler: Callable[[HomeAssistant, str, Request], Awaitable[Response | None]],
+    *,
```

Comment:
Out of curiosity, what is this * for?

Reply:
It is a delimiter to indicate all keyword-based arguments after this delimiter, arguments can only be passed in using their keywords (and not position-based) (See PEP-3102).

Reply:
I added this because the amount of positional arguments got crazy and I didn't want to add to it.